tools/run_tests/xds_k8s_test_driver/bin/black.sh

```diff
@@ -48,11 +50,11 @@ source "${XDS_K8S_DRIVER_DIR}/bin/ensure_venv.sh"
 
 if [[ "$1" == "--diff" ]]; then
   readonly MODE="--diff"
+elif [[ "$1" == "--check" ]]; then
+  readonly MODE="--check"
 else
-  readonly MODE="--in-place"
-  readonly VERBOSE="--verbose" # print out file names while processing
+  readonly MODE=""
```

Comment:
What will this default mode do?

Reply:
Black in the default mode (no args) will rewrite the code in place. `yapf` needed `--in-place` to do that, otherwise it'd just print the formatted file contents to the stdout.